src/test/common/utils.test.ts

```diff
@@ -10,21 +9,6 @@ describe('utils', () => {
 			assert.equal(utils.formatError(error), 'No!');
 		});
 
-		it('should format an HttpError from octorest', (done) => {
```

Comment:
This test was unreliable in CI because it relied on an actual network response. I couldn't find a different way to force Octokit to respond with an error without going into things like [nock](https://www.npmjs.com/package/nock) which felt redundant to the higher-level QueryProvider infra we already have, and we already test the branch that Octokit errors fall into in the previous case.